lib/tasks/convert_embeddables.rake

```diff
@@ -205,98 +176,131 @@ def convert()
     end
   end
 
-  if library_interactives.length == 0
-    raise Exception.new "No official library interactives found."
+  if library_interactives.length < 5
+    missing_official_interactives = @convertable_embeddable_types - library_interactives.keys
+    raise Exception.new "Error: Missing official library interactives. Before trying again, please mark an instance of the following library interactives as official: #{missing_official_interactives.join(", ")}"
   end
 
-  # activities_to_update = LightweightActivity.where(:runtime => "LARA")
-  activities_to_update = LightweightActivity.where(:id => 62)
+  activities_to_update = LightweightActivity.where(:runtime => "LARA")
   activities_to_update.each do |activity|
     activity.pages.each do |page|
       # Make a new ManagedInteractive version of each instance of the specified built-in
-      # embeddables and set the ManagedInteracative's legacy_ref_id to be the ref_id of the 
-      # old, built-in embeddable. For now, the new ManagedInteractive embeddable will not 
-      # be associated with the activity/page. That will be done by a different function.
+      # embeddables. Set the new ManagedInteracative's legacy_ref_id to be the ref_id of the 
+      # built-in embeddable.
       page.page_items.each do |item|
-        if item.embeddable # not sure what page items don't have embeddables - plugins?
+        library_interactive = nil
+        authored_state = nil
+        if item.embeddable
           legacy_ref_id = (LaraSerializationHelper.new).key(item.embeddable)
```

Comment:
This approach to computing the legacy_ref_id seems in efficient since it is creating a new LaraSerializationHelper for each embeddable. Could you re-use the same serialization helper object?